spec/text-editor-presenter-spec.coffee

```diff
@@ -832,11 +889,46 @@ describe "TextEditorPresenter", ->
         lineStateForScreenRow = (presenter, row) ->
           lineId  = presenter.model.tokenizedLineForScreenRow(row).id
           tileRow = presenter.tileForRow(row)
-          presenter.getState().content.tiles[tileRow]?.lines[lineId]
+          presenter.getState().content.tiles[tileRow]?.lines?[lineId]
 
         tiledContentContract (presenter) -> presenter.getState().content
 
+        it "contains the state for the longest tile on screen", ->
+          presenter = buildPresenter(explicitHeight: 4, scrollTop: 0, lineHeight: 1, tileSize: 2)
+
+          expectValues presenter.getState().content.tiles[6], {
+            visibility: "hidden"
+          }
```

Comment:
I am okay with removing them but this way the code feels a bit more consistent (i.e. the other specs use `expectValues` with curly brackets). 

What do you think if we just keep them for now?